test/system/setup_suite.bash

```diff
@@ -30,6 +30,60 @@ function setup_suite() {
     # The above does not handle errors. Do a final confirmation.
     assert "$PODMAN_LOGIN_REGISTRY_PORT" != "" \
            "Unable to set PODMAN_LOGIN_REGISTRY_PORT"
+
+    # Clean up all containers
```

Comment:
Instead of moving all this to suite, maybe define it as a function in helpers.bash, then invoke it from suite setup, then conditionally invoke it from setup() (I don't know what that condition looks like)